internal/transport/http2_client.go

```diff
@@ -1306,7 +1306,11 @@ func (t *http2Client) reader() {
 				if s != nil {
 					// use error detail to provide better err message
 					code := http2ErrConvTab[se.Code]
-					msg := t.framer.fr.ErrorDetail().Error()
+					errorDetail := t.framer.fr.ErrorDetail()
+					msg := ""
+					if errorDetail != nil {
+						msg = errorDetail.Error()
+					}
```

Comment:
Please add some text to `msg` when `errorDetail` is `nil`.

```go
	} else {
		msg = "unknown framer error"  // ??
	}
```

Reply:
I was unsure what comment is appropriate when errorDetail is unavailable...

Reply:
https://github.com/grpc/grpc-go/pull/3926/commits/c8eec21a0bbd611175b79f40c55528173df36ec8

Reply:
That sounds fine to me.